site/content/contribute/webapp/end-to-end-tests.md

```diff
@@ -18,6 +18,7 @@ Not familiar with Cypress? Here is some documentation that will help you get sta
 1.  Run a local server instance by initiating `make run` to the mattermost-server folder. Confirm that the instance has started successfully.
```

Comment:
If we're going to alter other parts of the instructions in this PR, I'd suggest:

```suggestion
1.  Launch a local Mattermost instance by running `make run` in the mattermost-server directory. Confirm that the Mattermost instance has started successfully.
```

Reply:
@justinegeffen ^

Reply:
@justinegeffen ping on this - would be good to get it merged.